scripts/dev_scripts/hooks/pre-push

```diff
@@ -14,4 +14,6 @@ local_branch_name=$(echo "$local_ref" | sed "s/refs\/heads\///")
 if [[ ($local_branch_name =~ ^__ || $local_branch_name =~ ^archived__ ) && $remote != "archive" ]]; then
   echo "Double underscore and archived branches can only be pushed to the archive (private) remote!"
   exit 1
-fi
\ No newline at end of file
+fi
+
+npm run check_format .
```

Comment:
Something to check, because I don't know off the top of my head, what's the working directory of a hook? The root of the git repo, or the location of the bash shell that ran the hooked command?

Reply:
It checks `client`, `email_backend`, and `server`, even when I'm in the `client` directory when I run the `git push`. So I'm guessing that it checks from root

Reply:
`check_format` is only defined in the root `package.json`, so if hooks weren't always run in the root context it'd be throwing errors instead.